modules/scheduler-impl/src/main/java/org/opencastproject/scheduler/impl/SchedulerServiceImpl.java

```diff
@@ -1667,9 +1654,7 @@ public String getCalendar(Opt<String> captureAgentId, Opt<String> seriesId, Opt<
         try {
           cal.addEvent(optMp.get(), catalogOpt.get(), agentId, start, end, lastModified, toPropertyString(caMetadata));
         } catch (Exception e) {
-          logger.warn("Error adding event '{}' to calendar, event is not recorded: {}", record.getMediaPackageId(),
-                  getStackTrace(e));
-          continue;
+          logger.warn("Error adding event '{}' to calendar, event is not recorded", record.getMediaPackageId(), e);
```

Comment:
The logger can handle stack traces by itself and continue is unnecessary since it's the end of the loop anyway.